### ISSUE TITLE
Pool of multiple workers using moviepy hangs eventually

### ISSUE THREAD

**Issue by cutecryptid** [Zulko/moviepy#1273]
I'm working on a render server that accepts video rendering tasks in JSON format and outputs MP4 videos (libx264) following a template.
Workers start fine and they render successfully, but eventually one after another ends up hanging when writing the final CompositeVideoFile. Checking other issues I've traced the problem to (probably) some pipe problem with FFMPEG but haven't managed to find a fix yet.
Each worker outputs files with unique filenames to a unique folder associated with the task, so there should be no problem with concurrent writing.
Tasks are issued from a Java application and through a ThreadPoolExecutor, but the actual tasks run entirely in the python render server. It shouldn't be a running out of memory issue either since the server that runs the system is processing two tasks at a time, and process manager shows sufficient free memory at all times.

``` 
v.write_videofile(out_path, codec=tCodec, fps=tFps, bitrate=tBitrate, logger=prog_logger,
               write_logfile=True, preset="superfast", temp_audiofile=tmp_audio_path)
v.close()
``` 

#### Expected Behavior
A set of ```multiprocessing.Process``` Objects that run moviepy tasks in parallel should render similar videos (different video sources but some common assets) without problem. 

#### Actual Behavior
The workers hang eventually, probably due to some ffmpeg issue, blocking the worker for future tasks and in the end leaving the whole system unable to process anything else. 

#### Steps to Reproduce the Problem
Launching multiple async moviepy processes with write_videofile result in the preovious issue. It's hard to reproduce the automated version of the problem since launching the process by hand yields different results.

#### Specifications
  - Python Version:
  - Moviepy Version: 1.0.3
  - Platform Name: Ubuntu
  - Platform Version: Ubuntu 20.04 LTS

####Logs

Good render:
``` 
ffmpeg version 4.2.2-static https://johnvansickle.com/ffmpeg/  Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 8 (Debian 8.3.0-6)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-debug --disable-ffplay --disable-indev=sndio --disable-outdev=sndio --cc=gcc --enable-fontconfig --enable-frei0r --enable-gnutls --enable-gmp --enable-libgme --enable-gray --enable-libaom --enable-libfribidi --enable-libass --enable-libvmaf --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librubberband --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libvorbis --enable-libopus --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libdav1d --enable-libxvid --enable-libzvbi --enable-libzimg
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
[rawvideo @ 0x6939880] Stream #0: not enough frames to estimate rate; consider increasing probesize
Input #0, rawvideo, from 'pipe:':
  Duration: N/A, start: 0.000000, bitrate: 49766 kb/s
    Stream #0:0: Video: rawvideo (RGB[24] / 0x18424752), rgb24, 1920x1080, 49766 kb/s, 1 tbr, 1 tbn, 1 tbc
Input #1, mp3, from '/storage/work/1-71-22/6731_1595577182_WsVzsEADBfXYJpjj_TEMP_MPY_wvf_snd.mp3':
  Metadata:
    encoder         : Lavf58.29.100
  Duration: 00:01:17.71, start: 0.025057, bitrate: 128 kb/s
    Stream #1:0: Audio: mp3, 44100 Hz, stereo, fltp, 128 kb/s
    Metadata:
      encoder         : Lavc58.54
Please use -b:a or -b:v, -b is ambiguous
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
  Stream #1:0 -> #0:1 (copy)
[libx264 @ 0x694c4c0] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2 AVX512
[libx264 @ 0x694c4c0] profile High, level 4.0, 4:2:0, 8-bit
[libx264 @ 0x694c4c0] 264 - core 159 r2991 1771b55 - H.264/MPEG-4 AVC codec - Copyleft 2003-2019 - http://www.videolan.org/x264.html - options: cabac=1 ref=1 deblock=1:0:0 analyse=0x3:0x3 me=dia subme=1 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=0 threads=6 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=1 keyint=250 keyint_min=1 scenecut=40 intra_refresh=0 rc=abr mbtree=0 bitrate=1024 ratetol=1.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 pb_ratio=1.30 aq=1:1.00
Output #0, mp4, to '/storage/work/1-71-22/6731_1595577182_WsVzsEADBfXYJpjj.mp4':
  Metadata:
    encoder         : Lavf58.29.100
    Stream #0:0: Video: h264 (libx264) (avc1 / 0x31637661), yuv420p, 1920x1080, q=-1--1, 1024 kb/s, 1 fps, 16384 tbn, 1 tbc
    Metadata:
      encoder         : Lavc58.54.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/1024000 buffer size: 0 vbv_delay: -1
    Stream #0:1: Audio: mp3 (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 128 kb/s
    Metadata:
      encoder         : Lavc58.54
frame=    2 fps=0.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    3 fps=3.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    3 fps=2.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    4 fps=2.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    4 fps=1.6 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    5 fps=1.6 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    5 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    6 fps=1.5 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    6 fps=1.3 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    7 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.3 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.3 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   11 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   12 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   13 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   14 fps=1.2 q=3.0 size=       0kB time=-00:00:00.02 bitrate=N/A speed=N/A    
frame=   15 fps=1.2 q=2.0 size=       0kB time=-00:00:00.02 bitrate=N/A speed=N/A    
frame=   16 fps=1.3 q=0.0 size=     768kB time=00:00:00.00 bitrate=5902288.9kbits/s speed=8.41e-05x    
frame=   17 fps=1.3 q=10.0 size=     768kB time=00:00:01.01 bitrate=6169.4kbits/s speed=0.0774x    
frame=   18 fps=1.3 q=11.0 size=    1024kB time=00:00:02.01 bitrate=4168.5kbits/s speed=0.147x    
frame=   19 fps=1.3 q=12.0 size=    1024kB time=00:00:03.00 bitrate=2791.5kbits/s speed=0.212x    
frame=   20 fps=1.4 q=3.0 size=    1792kB time=00:00:04.02 bitrate=3648.3kbits/s speed=0.274x    
frame=   21 fps=1.4 q=12.0 size=    2560kB time=00:00:05.01 bitrate=4180.5kbits/s speed=0.33x    
frame=   22 fps=1.4 q=14.0 size=    2560kB time=00:00:06.00 bitrate=3489.9kbits/s speed=0.383x    
frame=   24 fps=1.5 q=15.0 size=    2816kB time=00:00:08.02 bitrate=2876.2kbits/s speed=0.495x    
frame=   25 fps=1.5 q=16.0 size=    3072kB time=00:00:09.01 bitrate=2792.1kbits/s speed=0.539x    
frame=   26 fps=1.5 q=13.0 size=    3328kB time=00:00:10.00 bitrate=2724.7kbits/s speed=0.581x    
frame=   27 fps=1.5 q=17.0 size=    3328kB time=00:00:11.02 bitrate=2472.9kbits/s speed=0.622x    
frame=   28 fps=1.5 q=17.0 size=    3584kB time=00:00:12.01 bitrate=2443.2kbits/s speed=0.659x    
frame=   30 fps=1.6 q=18.0 size=    3584kB time=00:00:14.00 bitrate=2096.8kbits/s speed=0.747x    
frame=   31 fps=1.6 q=18.0 size=    3840kB time=00:00:15.02 bitrate=2094.2kbits/s speed=0.781x    
frame=   32 fps=1.6 q=20.0 size=    3840kB time=00:00:16.00 bitrate=1966.1kbits/s speed=0.81x    
frame=   33 fps=1.6 q=19.0 size=    4096kB time=00:00:17.00 bitrate=1973.8kbits/s speed=0.84x    
frame=   34 fps=1.6 q=21.0 size=    5120kB time=00:00:18.02 bitrate=2326.9kbits/s speed=0.868x    
frame=   35 fps=1.6 q=10.0 size=    5376kB time=00:00:19.01 bitrate=2315.7kbits/s speed=0.895x    
frame=   36 fps=1.7 q=21.0 size=    5632kB time=00:00:20.01 bitrate=2305.6kbits/s speed=0.919x    
frame=   37 fps=1.7 q=23.0 size=    5632kB time=00:00:21.00 bitrate=2196.7kbits/s speed=0.943x    
frame=   38 fps=1.7 q=22.0 size=    5888kB time=00:00:22.02 bitrate=2190.3kbits/s speed=0.967x    
frame=   39 fps=1.7 q=12.0 size=    6144kB time=00:00:23.01 bitrate=2186.9kbits/s speed=0.988x    
frame=   40 fps=1.7 q=22.0 size=    6656kB time=00:00:24.00 bitrate=2271.2kbits/s speed=1.01x    
frame=   41 fps=1.7 q=23.0 size=    6656kB time=00:00:25.00 bitrate=2181.0kbits/s speed=1.03x    
frame=   42 fps=1.7 q=24.0 size=    6912kB time=00:00:26.01 bitrate=2176.2kbits/s speed=1.05x    
frame=   44 fps=1.7 q=14.0 size=    7680kB time=00:00:28.00 bitrate=2246.6kbits/s speed=1.11x    
frame=   45 fps=1.7 q=23.0 size=    7680kB time=00:00:29.02 bitrate=2167.8kbits/s speed=1.12x    
frame=   46 fps=1.7 q=14.0 size=    7680kB time=00:00:30.01 bitrate=2096.1kbits/s speed=1.14x    
frame=   47 fps=1.8 q=23.0 size=    7936kB time=00:00:31.00 bitrate=2096.6kbits/s speed=1.16x    
frame=   48 fps=1.8 q=24.0 size=    7936kB time=00:00:32.00 bitrate=2031.6kbits/s speed=1.17x    
frame=   49 fps=1.8 q=25.0 size=    8448kB time=00:00:33.01 bitrate=2095.9kbits/s speed=1.19x    
frame=   50 fps=1.8 q=24.0 size=    8448kB time=00:00:34.01 bitrate=2034.7kbits/s speed= 1.2x    
frame=   51 fps=1.8 q=16.0 size=    8704kB time=00:00:35.00 bitrate=2036.9kbits/s speed=1.21x    
frame=   52 fps=1.8 q=25.0 size=    8960kB time=00:00:36.02 bitrate=2037.6kbits/s speed=1.23x    
frame=   53 fps=1.8 q=26.0 size=    8960kB time=00:00:37.01 bitrate=1982.9kbits/s speed=1.24x    
frame=   53 fps=1.7 q=26.0 size=    8960kB time=00:00:37.01 bitrate=1982.9kbits/s speed=1.22x    
frame=   54 fps=1.7 q=27.0 size=    8960kB time=00:00:38.00 bitrate=1931.1kbits/s speed=1.23x    
frame=   55 fps=1.8 q=17.0 size=    9216kB time=00:00:39.00 bitrate=1935.7kbits/s speed=1.24x    
frame=   55 fps=1.7 q=17.0 size=    9216kB time=00:00:39.00 bitrate=1935.7kbits/s speed=1.22x    
frame=   56 fps=1.7 q=24.0 size=    9472kB time=00:00:40.02 bitrate=1938.9kbits/s speed=1.24x    
frame=   58 fps=1.8 q=27.0 size=    9728kB time=00:00:42.00 bitrate=1897.2kbits/s speed=1.28x    
frame=   60 fps=1.8 q=24.0 size=   10240kB time=00:00:44.01 bitrate=1905.8kbits/s speed=1.32x    
frame=   62 fps=1.8 q=24.0 size=   10240kB time=00:00:46.00 bitrate=1823.5kbits/s speed=1.36x    
frame=   64 fps=1.9 q=27.0 size=   10496kB time=00:00:48.01 bitrate=1790.8kbits/s speed= 1.4x    
frame=   65 fps=1.9 q=25.0 size=   10752kB time=00:00:49.00 bitrate=1797.3kbits/s speed= 1.4x    
frame=   67 fps=1.9 q=25.0 size=   11008kB time=00:00:51.01 bitrate=1767.6kbits/s speed=1.44x    
frame=   67 fps=1.9 q=25.0 size=   11008kB time=00:00:51.01 bitrate=1767.6kbits/s speed=1.42x    
frame=   68 fps=1.9 q=27.0 size=   11008kB time=00:00:52.01 bitrate=1733.8kbits/s speed=1.43x    
frame=   68 fps=1.8 q=27.0 size=   11008kB time=00:00:52.01 bitrate=1733.8kbits/s speed=1.41x    
frame=   69 fps=1.8 q=19.0 size=   11264kB time=00:00:53.00 bitrate=1740.9kbits/s speed=1.42x    
frame=   69 fps=1.8 q=19.0 size=   11264kB time=00:00:53.00 bitrate=1740.9kbits/s speed= 1.4x    
frame=   71 fps=1.8 q=26.0 size=   11264kB time=00:00:55.00 bitrate=1677.7kbits/s speed=1.43x    
frame=   72 fps=1.9 q=-1.0 Lsize=   13407kB time=00:01:17.66 bitrate=1414.2kbits/s speed=2.01x    
video:12179kB audio:1214kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.108383%
[libx264 @ 0x694c4c0] frame I:19    Avg QP:10.61  size:345482
[libx264 @ 0x694c4c0] frame P:32    Avg QP:14.66  size:144875
[libx264 @ 0x694c4c0] frame B:21    Avg QP:19.99  size: 60479
[libx264 @ 0x694c4c0] consecutive B-frames: 52.8% 16.7% 25.0%  5.6%
[libx264 @ 0x694c4c0] mb I  I16..4: 17.3% 22.7% 60.0%
[libx264 @ 0x694c4c0] mb P  I16..4: 12.9% 28.9% 19.4%  P16..4: 27.0%  0.0%  0.0%  0.0%  0.0%    skip:11.8%
[libx264 @ 0x694c4c0] mb B  I16..4:  4.7% 13.0%  3.4%  B16..8: 26.7%  0.0%  0.0%  direct:25.9%  skip:26.3%  L0:42.7% L1:44.3% BI:12.9%
[libx264 @ 0x694c4c0] final ratefactor: 28.01
[libx264 @ 0x694c4c0] 8x8 transform intra:37.9% inter:25.9%
[libx264 @ 0x694c4c0] coded y,uvDC,uvAC intra: 82.2% 44.6% 25.1% inter: 34.3% 26.8% 4.8%
[libx264 @ 0x694c4c0] i16 v,h,dc,p: 53% 15% 20% 11%
[libx264 @ 0x694c4c0] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 13% 20% 23%  8%  6%  5%  9%  6% 11%
[libx264 @ 0x694c4c0] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 16% 29% 11%  8%  7%  5%  8%  5% 12%
[libx264 @ 0x694c4c0] i8c dc,h,v,p: 56% 22% 15%  7%
[libx264 @ 0x694c4c0] Weighted P-Frames: Y:21.9% UV:15.6%
[libx264 @ 0x694c4c0] kb/s:1385.58
``` 

Hanged-up render
``` 
ffmpeg version 4.2.2-static https://johnvansickle.com/ffmpeg/  Copyright (c) 2000-2019 the FFmpeg developers
  built with gcc 8 (Debian 8.3.0-6)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-debug --disable-ffplay --disable-indev=sndio --disable-outdev=sndio --cc=gcc --enable-fontconfig --enable-frei0r --enable-gnutls --enable-gmp --enable-libgme --enable-gray --enable-libaom --enable-libfribidi --enable-libass --enable-libvmaf --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-librubberband --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libvorbis --enable-libopus --enable-libtheora --enable-libvidstab --enable-libvo-amrwbenc --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libdav1d --enable-libxvid --enable-libzvbi --enable-libzimg
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
[rawvideo @ 0x6d97880] Stream #0: not enough frames to estimate rate; consider increasing probesize
Input #0, rawvideo, from 'pipe:':
  Duration: N/A, start: 0.000000, bitrate: 49766 kb/s
    Stream #0:0: Video: rawvideo (RGB[24] / 0x18424752), rgb24, 1920x1080, 49766 kb/s, 1 tbr, 1 tbn, 1 tbc
Input #1, mp3, from '/storage/work/1-71-4/2870_1595577182_pYpTIbmfUUdWmDjj_TEMP_MPY_wvf_snd.mp3':
  Metadata:
    encoder         : Lavf58.29.100
  Duration: 00:01:17.71, start: 0.025057, bitrate: 128 kb/s
    Stream #1:0: Audio: mp3, 44100 Hz, stereo, fltp, 128 kb/s
    Metadata:
      encoder         : Lavc58.54
Please use -b:a or -b:v, -b is ambiguous
Stream mapping:
  Stream #0:0 -> #0:0 (rawvideo (native) -> h264 (libx264))
  Stream #1:0 -> #0:1 (copy)
[libx264 @ 0x6daa4c0] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX FMA3 BMI2 AVX2 AVX512
[libx264 @ 0x6daa4c0] profile High, level 4.0, 4:2:0, 8-bit
[libx264 @ 0x6daa4c0] 264 - core 159 r2991 1771b55 - H.264/MPEG-4 AVC codec - Copyleft 2003-2019 - http://www.videolan.org/x264.html - options: cabac=1 ref=1 deblock=1:0:0 analyse=0x3:0x3 me=dia subme=1 psy=1 psy_rd=1.00:0.00 mixed_ref=0 me_range=16 chroma_me=1 trellis=0 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=0 threads=6 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=1 keyint=250 keyint_min=1 scenecut=40 intra_refresh=0 rc=abr mbtree=0 bitrate=1024 ratetol=1.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 pb_ratio=1.30 aq=1:1.00
Output #0, mp4, to '/storage/work/1-71-4/2870_1595577182_pYpTIbmfUUdWmDjj.mp4':
  Metadata:
    encoder         : Lavf58.29.100
    Stream #0:0: Video: h264 (libx264) (avc1 / 0x31637661), yuv420p, 1920x1080, q=-1--1, 1024 kb/s, 1 fps, 16384 tbn, 1 tbc
    Metadata:
      encoder         : Lavc58.54.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/1024000 buffer size: 0 vbv_delay: -1
    Stream #0:1: Audio: mp3 (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 128 kb/s
    Metadata:
      encoder         : Lavc58.54
frame=    2 fps=0.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    3 fps=3.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    3 fps=2.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    4 fps=2.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    4 fps=1.6 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    5 fps=1.6 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    5 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    6 fps=1.5 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    7 fps=1.5 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    7 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.4 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.3 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    8 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.3 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=    9 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   10 fps=1.0 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   11 fps=1.1 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   13 fps=1.2 q=0.0 size=       0kB time=00:00:00.00 bitrate=N/A speed=   0x    
frame=   14 fps=1.3 q=3.0 size=       0kB time=-00:00:00.02 bitrate=N/A speed=N/A    
frame=   15 fps=1.3 q=2.0 size=       0kB time=-00:00:00.02 bitrate=N/A speed=N/A    
frame=   16 fps=1.3 q=0.0 size=     768kB time=00:00:00.00 bitrate=5902288.9kbits/s speed=8.79e-05x    
frame=   17 fps=1.3 q=10.0 size=     768kB time=00:00:01.01 bitrate=6169.4kbits/s speed=0.0807x    
frame=   17 fps=1.3 q=10.0 size=     768kB time=00:00:01.01 bitrate=6169.4kbits/s speed=0.0776x    
frame=   18 fps=1.3 q=11.0 size=    1024kB time=00:00:02.01 bitrate=4168.5kbits/s speed=0.147x    
frame=   20 fps=1.4 q=3.0 size=    1792kB time=00:00:04.02 bitrate=3648.3kbits/s speed=0.284x    
frame=   21 fps=1.4 q=12.0 size=    2560kB time=00:00:05.01 bitrate=4180.5kbits/s speed=0.342x    
frame=   22 fps=1.4 q=14.0 size=    2560kB time=00:00:06.00 bitrate=3489.9kbits/s speed=0.396x    
frame=   23 fps=1.5 q=14.0 size=    2816kB time=00:00:07.00 bitrate=3294.7kbits/s speed=0.447x    
frame=   24 fps=1.5 q=15.0 size=    2816kB time=00:00:08.02 bitrate=2876.2kbits/s speed=0.496x    
frame=   26 fps=1.6 q=13.0 size=    3328kB time=00:00:10.00 bitrate=2724.7kbits/s speed= 0.6x    
frame=   27 fps=1.6 q=17.0 size=    3328kB time=00:00:11.02 bitrate=2472.9kbits/s speed=0.641x    
frame=   28 fps=1.6 q=17.0 size=    3584kB time=00:00:12.01 bitrate=2443.2kbits/s speed=0.679x    
frame=   29 fps=1.6 q=8.0 size=    3584kB time=00:00:13.01 bitrate=2256.8kbits/s speed=0.715x    
frame=   30 fps=1.6 q=18.0 size=    3840kB time=00:00:14.00 bitrate=2246.5kbits/s speed=0.749x    
frame=   31 fps=1.6 q=19.0 size=    3840kB time=00:00:15.02 bitrate=2094.2kbits/s speed=0.783x    
frame=   32 fps=1.6 q=20.0 size=    4096kB time=00:00:16.01 bitrate=2095.3kbits/s speed=0.813x    
frame=   33 fps=1.6 q=19.0 size=    4352kB time=00:00:17.00 bitrate=2096.3kbits/s speed=0.841x    
frame=   34 fps=1.6 q=21.0 size=    5120kB time=00:00:18.02 bitrate=2326.9kbits/s speed=0.87x    
frame=   35 fps=1.6 q=10.0 size=    5376kB time=00:00:19.01 bitrate=2315.7kbits/s speed=0.896x    
frame=   36 fps=1.7 q=21.0 size=    5632kB time=00:00:20.01 bitrate=2305.6kbits/s speed=0.921x    
frame=   38 fps=1.7 q=22.0 size=    6144kB time=00:00:22.02 bitrate=2285.5kbits/s speed=0.991x    
frame=   39 fps=1.7 q=12.0 size=    6144kB time=00:00:23.01 bitrate=2186.9kbits/s speed=1.01x    
frame=   40 fps=1.7 q=22.0 size=    6656kB time=00:00:24.00 bitrate=2271.2kbits/s speed=1.03x    
frame=   42 fps=1.8 q=22.0 size=    6912kB time=00:00:26.01 bitrate=2176.2kbits/s speed= 1.1x    
frame=   43 fps=1.8 q=25.0 size=    7168kB time=00:00:27.01 bitrate=2173.9kbits/s speed=1.11x    
frame=   44 fps=1.8 q=14.0 size=    7680kB time=00:00:28.00 bitrate=2246.6kbits/s speed=1.13x    
frame=   46 fps=1.8 q=14.0 size=    7936kB time=00:00:30.01 bitrate=2165.9kbits/s speed=1.19x    
frame=   47 fps=1.8 q=23.0 size=    7936kB time=00:00:31.00 bitrate=2096.6kbits/s speed= 1.2x    
frame=   48 fps=1.8 q=24.0 size=    7936kB time=00:00:32.00 bitrate=2031.6kbits/s speed=1.22x    
frame=   49 fps=1.8 q=25.0 size=    8448kB time=00:00:33.01 bitrate=2095.9kbits/s speed=1.23x    
frame=   51 fps=1.9 q=16.0 size=    8960kB time=00:00:35.00 bitrate=2096.9kbits/s speed=1.28x    
frame=   52 fps=1.9 q=25.0 size=    8960kB time=00:00:36.02 bitrate=2037.6kbits/s speed= 1.3x    
frame=   53 fps=1.9 q=27.0 size=    9216kB time=00:00:37.01 bitrate=2039.6kbits/s speed=1.31x    
frame=   54 fps=1.9 q=24.0 size=    9216kB time=00:00:38.00 bitrate=1986.3kbits/s speed=1.32x    
frame=   55 fps=1.9 q=17.0 size=    9472kB time=00:00:39.00 bitrate=1989.5kbits/s speed=1.33x    
frame=   56 fps=1.9 q=24.0 size=    9728kB time=00:00:40.02 bitrate=1991.3kbits/s speed=1.34x    
frame=   56 fps=1.8 q=24.0 size=    9728kB time=00:00:40.02 bitrate=1991.3kbits/s speed=1.32x    
frame=   57 fps=1.8 q=25.0 size=    9984kB time=00:00:41.01 bitrate=1994.2kbits/s speed=1.33x    
frame=   57 fps=1.8 q=25.0 size=    9984kB time=00:00:41.01 bitrate=1994.2kbits/s speed=1.31x    
frame=   57 fps=1.8 q=-1.0 Lsize=   11919kB time=00:01:17.66 bitrate=1257.2kbits/s speed=2.44x    
video:10690kB audio:1214kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 0.118259%
[libx264 @ 0x6daa4c0] frame I:13    Avg QP: 8.44  size:417342
[libx264 @ 0x6daa4c0] frame P:28    Avg QP:14.16  size:152750
[libx264 @ 0x6daa4c0] frame B:16    Avg QP:18.79  size: 77727
[libx264 @ 0x6daa4c0] consecutive B-frames: 54.4% 14.0% 31.6%  0.0%
[libx264 @ 0x6daa4c0] mb I  I16..4: 16.1% 19.0% 64.9%
[libx264 @ 0x6daa4c0] mb P  I16..4: 13.5% 30.6% 19.7%  P16..4: 27.1%  0.0%  0.0%  0.0%  0.0%    skip: 9.1%
[libx264 @ 0x6daa4c0] mb B  I16..4:  4.5% 14.0%  5.1%  B16..8: 26.7%  0.0%  0.0%  direct:26.2%  skip:23.5%  L0:39.9% L1:46.3% BI:13.8%
[libx264 @ 0x6daa4c0] final ratefactor: 28.06
[libx264 @ 0x6daa4c0] 8x8 transform intra:38.3% inter:25.3%
[libx264 @ 0x6daa4c0] coded y,uvDC,uvAC intra: 82.9% 44.5% 26.8% inter: 39.3% 29.0% 6.3%
[libx264 @ 0x6daa4c0] i16 v,h,dc,p: 51% 17% 21% 11%
[libx264 @ 0x6daa4c0] i8 v,h,dc,ddl,ddr,vr,hd,vl,hu: 14% 20% 22%  7%  6%  5%  9%  5% 11%
[libx264 @ 0x6daa4c0] i4 v,h,dc,ddl,ddr,vr,hd,vl,hu: 17% 29% 11%  7%  6%  5%  8%  5% 11%
[libx264 @ 0x6daa4c0] i8c dc,h,v,p: 57% 22% 14%  7%
[libx264 @ 0x6daa4c0] Weighted P-Frames: Y:25.0% UV:17.9%
[libx264 @ 0x6daa4c0] kb/s:1536.29
```

**Comment by gergovari** [comment]
Same issue, @cutecryptid have you found any workarounds?

**Comment by OsaAjani** [comment]
Thank you for your contributions and for reporting issues in this repository. With the release of v2, which introduces significant changes to the codebase and API, we’ve reviewed the backlog of open PRs and issues. Due to the length of the backlog and the likelihood that many of these are either fixed or no longer applicable, we’ve made the decision to close all previous PRs and issues.

If you believe that any of these are still relevant to the current version or if you'd like to reopen a related discussion, please feel free to create a new issue or pull request, referencing the old one.

Thank you for your understanding and continued support!
